docs/contributing/documentation.md

```diff
@@ -0,0 +1,9 @@
+# Writing Docs for orchestrator-core
+
+## Setting up Documentation Environment
+
+1. Install flit `pip install flit` inside of the venv you will use for orch development.
+1. setup dev env for writing docs `flit install`
+1. Write some documentation by editing the markdown files in `orchestrator-core/docs/`
+1. Build the documentation with `mkdocs build`.
+1. View the documentation by opening `orchestrator-core/site/index.html` in your browser.
```

Comment:
it is also possible to use `mkdocs serve`

Reply:
Good call, this is FAR more convenient!